packages/components/src/FormField/FormField.test.tsx

```diff
@@ -201,7 +201,7 @@ describe("FormField", () => {
         );
 
         expect(validationHandler).toHaveBeenCalled();
-        expect(validationHandler).toHaveBeenCalledWith(undefined);
+        expect(validationHandler).toHaveBeenCalledWith("");
```

Comment:
The name of the test should be updated. It should no longer be `should trigger onValidation with undefined`